java-checks/src/test/files/checks/CatchUsesExceptionWithContextCheck.java

```diff
@@ -93,6 +93,7 @@ private void h() {
       /* do nothing */
     } catch (ParseException e) {                 // Compliant
     } catch (MalformedURLException e) {          // Compliant
+    } catch (DateTimeParseException e) {          // Compliant
```

Comment:
Use the fully qualified name for the exception (`java.time.format.DateTimeParseException`). As you didn't added the exception to the imports, the analyzer won't be able to resolve its type and will consider it as unknown... and unknown exception types do not raise issues.